### PR TITLE
Fixes #20905 - Add message in audit in case no changes 

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

**Comment by shiramax** [comment]
Added the following message in audit : 
image

@Rohoover , what do you think about the design? thanks.

**Comment by ares** [comment]
There was a recent improvement no template history tab too, see #4773 I think it would be good to display it the same way. 
template

**Comment by shiramax** [comment]
@ares , 
Is this better? 
image

**Comment by Rohoover** [comment]
bitmoji

**Review by ares — CHANGES_REQUESTED** [review]
It's much better but now I think it does not behave correctly. See the screenshot for new template and update of existing one when I changed name and template content at the same time

tmp1
tmp2

**Comment by shiramax** [comment]
@ares, yes I think I fixed it now, please let me know what you think.

**Review by tbrisker — COMMENTED** [review]
minor nitpick - looks like a large part of the diff is an indentation change making it more difficult to see what changed. Could you please make sure you stay with the indentation level we use everywhere (2 spaces per level)?

File: app/views/audits/show.html.erb [hunk 1]
```diff
@@ -23,42 +23,44 @@
...
-            <th><%= _('Item') %></th>
-            <% if @audit.action == 'update' %>
-              <th><%= _('Old') %></th>
-              <th><%= _('New') %></th>
-            <% else %>
-              <th><%= _('Value') %></th>
-            <% end  %>
-          </tr>
-        </thead>
+        <%if @audit.audited_changes.key?("template") && @audit.audited_changes.length==1 %>
```
Comment (ares) [inline app/views/audits/show.html.erb]: please add a space after `<%` and around `==`, also I think `@audit.audited_changes.size` is prefered

**Review by ares — CHANGES_REQUESTED** [review]
it now works great, one comment in the code and then good to go

**Comment by shiramax** [comment]
@ares, Done :)

**Review by ares — APPROVED** [review]

**Comment by ares** [comment]
Works just great, the indentation now looks correct. Thanks @shiramax, merged as d79cc18d1c5bd5d71d4b2a22370c5254635cae27

**Pull request closed without merging**